src/main/scala/com/vngrs/etl/Loader.scala

```diff
@@ -0,0 +1,31 @@
+package com.vngrs.etl
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.Row
+
+trait Loader[A] {
+
+  protected[etl] def apply(input: RDD[A]): Loader[A]
```

Comment:
Yeah could be.
But load operation is not defined on `Loader` trait so we can't do it right now 😢